packages/client/components/TeamPrompt/WorkDrawer/GitHubObjectCard.tsx

```diff
@@ -0,0 +1,140 @@
+        __typename
+        ... on _xGitHubIssue {
+          id
+          title
+          number
+          repository {
+            nameWithOwner
+            url
+          }
+          url
+          issueState: state
+          lastEvent: timelineItems(last: 1) {
+            updatedAt
+          }
+        }
+        ... on _xGitHubPullRequest {
+          id
+          title
+          number
+          repository {
+            nameWithOwner
+            url
+          }
+          url
+          pullRequestState: state
+          lastEvent: timelineItems(last: 1) {
+            updatedAt
+          }
+          isDraft
+        }
+      }
+    `,
+    resultRef
+  )
+
+  const {tooltipPortal, openTooltip, closeTooltip, originRef} = useTooltip<HTMLDivElement>(
+    MenuPosition.UPPER_CENTER
+  )
+
+  if (result?.__typename !== '_xGitHubIssue' && result?.__typename !== '_xGitHubPullRequest') {
+    return null
+  }
+
+  const {title, number, repository, url, lastEvent} = result
+  const {nameWithOwner: repoName, url: repoUrl} = repository
+  const {updatedAt} = lastEvent
+  const status =
+    result?.__typename === '_xGitHubIssue' ? result.issueState : result.pullRequestState
+  const prIsDraft = result?.__typename === '_xGitHubPullRequest' && result.isDraft
+
+  const modifiedStatus = prIsDraft && status === 'OPEN' ? 'DRAFT' : status
+  const statusImg =
+    result.__typename === '_xGitHubIssue'
+      ? ISSUE_STATUS_MAP[modifiedStatus]
+      : PR_STATUS_MAP[modifiedStatus]
+
+  return (
+    <div className='rounded border border-solid border-slate-300 p-4 hover:border-slate-600'>
+      <div className='flex gap-2 text-xs text-slate-600'>
+        {statusImg && <img src={statusImg} />}
+        <a href={url} target='_blank' className='font-medium hover:underline' rel='noreferrer'>
+          #{number}
+        </a>
+        <div>Updated {relativeDate(updatedAt)}</div>
+      </div>
+      <div className='my-2 text-sm'>
+        <a href={url} target='_blank' className='hover:underline' rel='noreferrer'>
+          {title}
+        </a>
+      </div>
+      <div className='flex items-center justify-between'>
+        <div className='flex items-center gap-2'>
+          <div className='h-4 w-4'>
+            <img src={gitHubSVG} />
+          </div>
+          <a
+            href={repoUrl}
+            target='_blank'
+            className='text-xs text-slate-600 hover:underline'
+            rel='noreferrer'
+          >
+            {repoName}
+          </a>
+        </div>
+        <CopyToClipboard text={url}>
+          <div
+            className='h-6 rounded-full bg-transparent p-0 text-slate-500 hover:bg-slate-200'
+            onMouseEnter={openTooltip}
+            onMouseLeave={closeTooltip}
+            ref={originRef}
+          >
+            <Link className='h-6 w-6 cursor-pointer p-0.5' />
+          </div>
+        </CopyToClipboard>
+        {tooltipPortal('Copy link')}
```

Comment:
+1 in our invitation modal we use the same icon & after they click the link we pop a tooltip saying "Copied!" to affirm that the data is copied to clipboard. Could we reuse that pattern here?

Reply:
Added!